scripts/publish.sh

```diff
@@ -19,10 +19,16 @@ echo "============================"
 RELEASE_CHANNEL=$1
 echo "RELEASE_CHANNEL: $RELEASE_CHANNEL"
 
-NPM_VERSION=$2
+PRISMA_VERSION=$(sh scripts/prisma-version.sh "$CHANNEL")
+echo "PRISMA_VERSION: $PRISMA_VERSION"
+
+NPM_VERSION=$(sh scripts/prisma-version.sh "$CHANNEL")
 echo "NPM_VERSION: $NPM_VERSION"
```

Comment:
Wouldn't `PRISMA_VERSION` and `NPM_VERSION` variables hold the same value as they are filled from the same script?

Reply:
yes, I should remove the `PRISMA_VERSION` here since it is not used in this script.